compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/Serializability.hs

```diff
@@ -102,6 +102,10 @@ serializabilityConditionsType world0 _version mbModNameTpls vars = go
         BTNumeric -> Left URNumeric -- 'Numeric' is used as a higher-kinded type constructor.
         BTAny -> Left URAny
         BTTypeRep -> Left URTypeRep
+        BTAnyException -> noConditions
+        BTGeneralError -> noConditions
+        BTArithmeticError -> noConditions
+        BTContractError -> noConditions
```

Comment:
We need to check that all the fields of exception types are serializable too.

Reply:
Yes. Separate PR!